com.dynamo.cr/com.dynamo.cr.bob/scripts/copy.sh

```diff
@@ -82,7 +82,7 @@ cp -v $DYNAMO_HOME/ext/bin/x86_64-linux/luajit-32 libexec/x86_64-linux/luajit-32
 cp -v $DYNAMO_HOME/ext/bin/x86_64-linux/luajit-64 libexec/x86_64-linux/luajit-64
 cp -v $DYNAMO_HOME/ext/bin/x86_64-macos/luajit-32 libexec/x86_64-macos/luajit-32
 cp -v $DYNAMO_HOME/ext/bin/x86_64-macos/luajit-64 libexec/x86_64-macos/luajit-64
-#cp -v $DYNAMO_HOME/ext/bin/arm64-macos/luajit-32 libexec/arm64-macos/luajit-32 # TODO create an arm32 executable
+cp -v $DYNAMO_HOME/ext/bin/arm64-macos/luajit-32 libexec/arm64-macos/luajit-32 # TODO create an arm32 executable
```

Comment:
Add a luajit-32 into the arm64-macos folder (to be able to build for 32 bit targets)